harbor-ui/src/routes/receive.rs

```diff
@@ -24,10 +25,15 @@ pub fn receive(harbor: &HarborWallet) -> Element<Message> {
 
 /// Renders the view before an invoice/address is generated.
 fn render_receive_form(harbor: &HarborWallet) -> Element<Message> {
-    let header = if harbor.onchain_receive_enabled
+    // for on-chain to be shown, it needs to be a federation and either
+    // the user turned on on-chain receive or the federation supports it
+    let header = if matches!(
```

Comment:
Not sure this logic is working I'm seeing on-chain receive for all my cashu mints when I have on-chain turned on

Reply:
Looks like the logic was broken before too! Fixed now